stories/Atom/Images/Authorimage/Small.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './authorimage.scss';
+import user from '../../../assets/images/user.svg';
+
+export const Small = ({ src, alt }) => {
+    const image = {
+        src: user,
+        alt: 'user',
+    }
+    return (
+        <div className="author__img__small">
```

Comment:
@Shubham-Sharma-77 Here keep the author__img class and for the modifier add seperate class author__img--small and overwrite the styles on this